rasa/core/policies/policy.py

```diff
@@ -147,18 +151,25 @@ def create(
         """Creates a new untrained policy (see parent class for full docstring)."""
         return cls(config, model_storage, resource, execution_context)
 
-    @classmethod
-    def _create_featurizer(cls, policy_config: Dict[Text, Any]) -> TrackerFeaturizer:
+    def _create_featurizer(self, policy_config: Dict[Text, Any]) -> TrackerFeaturizer:
         policy_config = copy.deepcopy(policy_config)
 
         featurizer_configs = policy_config.get("featurizer")
 
         if not featurizer_configs:
-            return cls._standard_featurizer()
+            return self._standard_featurizer()
+
+        if (
+            POLICY_MAX_HISTORY not in featurizer_configs[0]
+            and POLICY_MAX_HISTORY in policy_config
+        ):
+            featurizer_configs[0][POLICY_MAX_HISTORY] = policy_config[
+                POLICY_MAX_HISTORY
+            ]
```

Comment:
how about passing the `max_history` separately to `_get_featurizer_from_config`? This would avoid weird errors in case the `featurizer_configs` are not a list

Reply:
I was wondering why featurizer configs is a list at all?

Reply:
no clue 🤷🏻  I guess some historic reason. I've just moved the code for this but didn't do any changes to the current config